gn2/wqflask/templates/wiki/history.html

```diff
@@ -0,0 +1,141 @@
+{% extends "base.html" %}
+{% block css %}
+    <style type="text/css">
+        h3 {
+            color: #336699;
+        }
+
+        table tbody th {
```

Comment:
can we try to use grid-adjacent elements here instead?  e.g. we can add the table into a div to enforce this width, something like:
```
<div class="row">
   <div class="col-sm-9">
   <table ...>
  </div>
</div>
```